spec/context-loss.md

```diff
@@ -34,14 +34,18 @@ When the user agent detects that the backing storage associated with a Canvas2D
     1. Fire a event `contextlost` at *canvas*
     2. If the event's canceled flag is not set, abort these steps.
```

Comment:
Oh, I think this logic is backwards (from what's in the code, and what has been discussed previously) and needs to be changed. (2D context behavior is opposite the behavior of WebGL context lost for this case.): So `If the event's canceled flag is set, abort these steps` and then add some kind of a description around not restoring the canvas (and what that means).